lib/src/screens/receive/receive_page.dart

```diff
@@ -122,6 +123,9 @@ class ReceivePage extends BasePage {
                     amountController: _amountController,
                     isLight: currentTheme.type == ThemeType.light),
               ),
+              AddressList(
+                addressListViewModel: addressListViewModel,
+              ),
```

Comment:
you added this without removing the old one, so now they are 2 in the UI
<img width="345" alt="Screenshot 2024-08-15 at 3 56 24 AM" src="https://github.com/user-attachments/assets/49d875f6-e779-43f7-8c86-865915866e17">